packages/manager/apps/pci-users/src/__tests__/components/S3StorageRegions.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import 'element-internals-polyfill';
+import { describe, vi } from 'vitest';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import queryClient from '@/queryClient';
+import StorageRegions from '@/components/users/StorageRegions';
+import { Region } from '@/data/region';
+import S3StorageRegions from '@/components/users/S3StorageRegions.tsx';
+
+vi.mock('@ovh-ux/manager-react-shell-client', async () => ({
+  useEnvironment: () => ({
+    user: {},
+  }),
+}));
+
+vi.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translation hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str: string) => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {}),
+      },
+    };
+  },
+  initReactI18next: {
+    type: '3rdParty',
+    init: () => {},
+  },
+}));
+
+vi.mock('@/hooks/useRegion', () => {
+  const regions: Region[] = [
+    {
+      name: 'region',
+      type: 'type',
+      services: [],
+      datacenterLocation: '',
+      ipCountries: [],
+      status: '',
+      continentCode: '',
+    },
+  ];
+  return {
+    useS3StorageRegions: () => ({
+      data: regions,
+    }),
+  };
+});
+
+function renderComponent() {
+  render(
+    <QueryClientProvider client={queryClient}>
+      <S3StorageRegions
+        projectId="foo"
+        onS3StorageRegionChange={() => {}}
+      ></S3StorageRegions>
+      ,
+    </QueryClientProvider>,
+  );
+}
+
+describe('S3StorageRegions components', () => {
+  it('should call the S3StorageRegions component', async () => {
+    renderComponent();
```

Comment:
it looks like you are doing a render but not checking any value, so this test will always pass
what do you want to test ?